src/generator/common/helpers.ts

```diff
@@ -135,13 +138,13 @@ export function formatParamInfoTypeName(param: ParamInfo): string {
 
 // aggregates the Parameter in op.parameters and the first request
 export function aggregateParameters(op: Operation): Array<Parameter> {
-  if (op.requests!.length > 1) {
-    throw console.error('multiple requests NYI');
-  }
   let params = new Array<Parameter>();
   if (op.parameters) {
     params = params.concat(op.parameters);
   }
+  if (op.requests!.length > 1) {
```

Comment:
Nit: move this block back to line 138 so diff goes away.

Reply:
You mean line 141? outside of that it would be outside of the functions

Reply:
Moved it